File: app/src/main/java/com/seafile/seadroid2/framework/data/model/sdoc/SDocCommentReplyModel.java
package com.seafile.seadroid2.framework.data.model.sdoc;

import com.google.gson.annotations.JsonAdapter;
import com.seafile.seadroid2.framework.data.model.adapter.OffsetDateTimeAdapter;

import java.time.OffsetDateTime;

public class SDocCommentReplyModel {
    public int id;
    public int comment_id;

    public String doc_uuid;
    public String reply;//"True"

    public String author;
    public String avatar_url;
    public String type;//"type"

    public String resolved;
    public String user_contact_email;
    public String user_email;
    public String user_name;


    @JsonAdapter(OffsetDateTimeAdapter.class)
    public OffsetDateTime created_at;

    @JsonAdapter(OffsetDateTimeAdapter.class)
    public OffsetDateTime updated_at;

}

File: app/src/main/java/com/seafile/seadroid2/framework/model/sdoc/SDocCommentReplyModel.kt
package com.seafile.seadroid2.framework.model.sdoc

import com.google.gson.annotations.JsonAdapter
import com.seafile.seadroid2.framework.model.adapter.OffsetDateTimeAdapter
import java.time.OffsetDateTime

class SDocCommentReplyModel {
    @JvmField
    var id: Int = 0

    @JvmField
    var comment_id: Int = 0

    @JvmField
    var doc_uuid: String? = null

    @JvmField
    var reply: String? = null

    @JvmField
    var author: String? = null

    @JvmField
    var avatar_url: String? = null

    @JvmField
    var type: String? = null

    @JvmField
    var resolved: String? = null

    @JvmField
    var user_contact_email: String? = null

    @JvmField
    var user_email: String? = null

    @JvmField
    var user_name: String? = null

    @JsonAdapter(OffsetDateTimeAdapter::class)
    @JvmField
    var created_at: OffsetDateTime? = null

    @JsonAdapter(OffsetDateTimeAdapter::class)
    @JvmField
    var updated_at: OffsetDateTime? = null
}
